includes/utilities.inc

```diff
@@ -946,3 +946,25 @@ function islandora_deployed_on_windows() {
   }
   return FALSE;
 }
+
+/**
+ * Build the edit registry for a given datastream.
+ *
+ * @param AbstractObject $object
+ *   The object being edited.
+ * @param AbstractDatastream $datastream
+ *   The datastream being edited.
+ *
+ * @return array
+ *   The built edit registry array.
+ */
+function islandora_build_datastream_edit_registry($object, $datastream) {
```

Comment:
`$object` doesn't actually have to be passed in here. I think it would be safer to just have something like:
```php
$object = $datastream->parent;
```
below, to avoid the potential of somebody passing in unrelated objects.